site/content/en/docs/admission-check-controllers/multikueue.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "MultiKueue Admission Check Controller"
+date: 2024-02-08
+weight: 2
+description: >
+  An admission check controller providing the manager side MultiKueue functionality.
+---
+
+The MultiKueue Admission Check Controller is an Admission Check Controller designed to provide the manager side functionality for multi cluster job dispatching.
+
+You can enable it by setting the `MultiKueue` feature gate. Check the [Installation](/docs/installation/#change-the-feature-gates-configuration) guide for details on feature gate configuration.
+
+{{% alert title="Warning" color="warning" %}}
+MultiKueue is currently an alpha feature and disabled by default.
+{{% /alert %}}
+
+## MultiKueue Overview
+
+A MultiKueue setup is composed of a manager cluster and at least one worker cluster.
+
+### Cluster Roles
+#### Manager Cluster
+
+The manager's main attributes are:
+- Establish and maintain the connection with the worker clusters.
+- Create and monitor remote objects (workloads or jobs) while keeping the local ones in sync.
+
+The MultiKueue Admission Check Controller runs in the manager cluster and will also maintain the `Active` status of the Admission Checks controlled by `multikueue`.
+
+The ResourceQuota set on a ClusterQueue using MultiKueue controls how many jobs are subject for dispatching at a given point in time, ideally it should be less then the total quotas in the worker clusters since otherwise will cause the monitoring of workloads that don't have a chance to be admitted. 
+
+#### Worker Cluster
+
+The worker cluster acts like an usual standalone Kueue cluster in which the Kueue the workloads and jobs are managed by the MultiKueue Admission Check Controller running in the manager cluster.
+
+### Job Flow
+
+If the ClusterQueue a job is assigned to in the manager cluster uses a MultiKueue AdmissionCheck it will be the subject of dispatching, as a result:
+
+- When it's workload gets a QuotaReservation in the manager, a copy of that workload will be created in all the configured worker clusters.
+- When one of the remote workloads is admitted in a worker cluster:
+  - All the other remote workloads are removed.
+  - A copy of the job is created in the selected worker cluster having the remote workload set as its prebuilt-workload.
```

Comment:
Can we mention the prebuild annotation here?

Reply:
done